docs/chitter_repository_class_design.md

```diff
@@ -0,0 +1,283 @@
+# Albums Model and Repository Classes Design Recipe
```

Comment:
Good shout of added the .md files to the docs folder! It keeps the folder stucture clean.